cbcontainers/remote_configuration/change_applier.go

```diff
@@ -78,3 +79,17 @@ func toggleFeaturesBasedOnCompatibility(cr *cbcontainersv1.CBContainersAgent, ve
 		cr.Spec.Components.Cndr.Enabled = &falseRef
 	}
 }
+
+func applyAdvancedSettings(settings *models.AdvancedSettings, cr *cbcontainersv1.CBContainersAgent) {
```

Comment:
Add some tests for this path (that it gets applied; that it gets ignored at least)